### DISCUSSION TITLE
How to publish a site after the site being unpublished

### DISCUSSION THREAD

**Discussion by Shuggy999** [community/community#32427] [Other Feature Feedback, Questions, & Ideas]
Hello! 
Does anyone know how to publish a site after the site being unpublished?
I accidently unpublished my site and I do not know how to get to back online.

**Comment by radar-blip** [comment]
Did you fix this or still having problems?

**Reply by Shuggy999** [reply]
> Did you fix this or still having problems?

Yes - sorry I didn't see your reply

**Reply by radar-blip** [reply]
Thats fine. Glad you got it fixed!
